README.md

```diff
@@ -93,52 +129,51 @@ This RFC makes no requirement as to how the transition should happen, but here a
+  - `package.nix`/`pkg.nix`: Makes the migration to a non-function form of overridable packages harder in the future.
+  - `default.nix`:
+    - Doesn't have its main benefits in this case:
+      - Removing the need to specify the file name in expressions, but this does not apply because this file will be imported automatically by the code that replaces definitions from `all-packages.nix`.
+      - Removing the need to specify the file name on the command line, but this does not apply because a package function must be imported into an expression before it can be used, making `nix build -f pkgs/unit/hell/hello` equally broken regardless of file name.
+    - Not using `default.nix` frees up `default.nix` for a short expression that is actually buildable, e.g. `(import ../..).hello`.
+    - Choosing `default.nix` would bias the purpose of the `unit` directory to serve only as package definitions, whereas we could make the tree more human friendly by grouping files together by "topic" rather than by technical delineations.
+      For instance, having a package definition, changelog, package-specific config generator and perhaps even NixOS module in one directory makes work on the package in a broad sense easier.
+      This is not a goal of this RFC, but a motivation to make this a future possibility.
+
+- Use `unit` (at the nixpkgs root) instead of `pkgs/unit`.
+  This is future proof in case we want to make the directory structure more general purpose, but this is out of scope
+- Additionally have a backwards-compatibility layer for moved paths, such as a symlink pointing from the old to the new location, or for Nix files even a `builtins.trace "deprecated" (import ../new/path)`.
+  We are not doing this because it would give precedent to file paths being a stable API interface, which definitely shouldn't be the case (bar some exceptions).
+  It would also lead to worse merge conflicts as the transition is happening, since Git would have to resolve a merge conflict between a symlink and a changed file.
+- Loosen [criteria 3](#user-content-criteria-3), allowing certain packages to be moved to the new structure even if it requires updating references to paths in Nix files.
+  This isn't done because it [turns out](https://github.com/nixpkgs-architecture/simple-package-paths/issues/14) that this criteria indicates the file structure being used as an API interface.
+  By manually refactoring the Nix code to not rely on this anymore, you can increase code quality/reusability/clarity and then do the transition described in the RFC.
+- Use a different sharding scheme than `<4-prefix name>`.
+  Discussions regarding this can be seen [here](https://github.com/nixpkgs-architecture/simple-package-paths/issues/1), [NAT meeting #18](https://github.com/nixpkgs-architecture/meetings/blob/6282b0c6bbc47b6f1becd155586c79728eddefc9/2022-11-21.md) and [here](https://github.com/nixpkgs-architecture/simple-package-paths/pull/20#discussion_r1029004083)
```

Comment:
Should be summarized here.

Reply:
I redid the alternatives section, including writing a summary of this one here